daprdocs/content/en/operations/components/setup-pubsub/_index.md

```diff
@@ -4,4 +4,53 @@ title: "Pub/Sub brokers"
 linkTitle: "Pub/sub brokers"
 description: "Guidance on setting up different message brokers for Dapr Pub/Sub"
 weight: 2000
+aliases: 
+  - "/operations/components/setup-pubsub/setup-pubsub-overview/"
 ---
+
+Dapr integrates with pub/sub message buses to provide applications with the ability to create event-driven, loosely coupled architectures where producers send events to consumers via topics. 
```

Comment:
Is this a new file or a rename that git doesn't recognize?

Reply:
A dedicated overview page didn't make sense for pub/sub so I moved the content from the existing overview page to the index page.